server/apps/highlights/service.py

```diff
@@ -72,17 +78,27 @@ def create(self, docs, **kwargs):
             highlights = item.get('highlights', [])
             if not highlights:
```

Comment:
Actually it isn't, `highlights` can also exist in the item dictionary and be equal to `None`, hence the need for the additional check.